admin/src/js/controllers/main.js

```diff
@@ -20,6 +20,8 @@ angular.module('controllers').controller('MainCtrl',
       $window.location.href = Location.path;
     });
 
+
+    $scope.baseUrl = Location.path;
```

Comment:
`baseUrl` is a bit misleading because it's not the base of _this_ app. Can we change it to `webappUrl` or something?